PlaywrightSolar/PageObjects/InvertersPage.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightSolar.PageObjects;
+
+public class InvertersPage(IPage page)
+{
+    // Locators
+    private ILocator InvertersPageTitle =>
+        page.GetByRole(AriaRole.Heading, new PageGetByRoleOptions { Name = "Сонячні інвертори" });
+
+    private ILocator PlaceOrderButton =>
+        page.GetByRole(AriaRole.Link, new PageGetByRoleOptions { Name = "Оформити замовлення" });
+
+
+    // Methods
+    public async Task VerifyInvertersPageTitle()
+    {
+        await Task.Delay(1500);
+        await Assertions.Expect(InvertersPageTitle).ToBeVisibleAsync();
+    }
+
+    public async Task<string> AddProductToBasket()
+    {
+        var listOfProductsOnPage = await page.QuerySelectorAllAsync(".col.s12.m6.l4.xl3.prod-holder");
+        var random = new Random();
+        var randomIndex = random.Next(listOfProductsOnPage.Count);
+        var selectedProductFromPage = listOfProductsOnPage[randomIndex];
```

Comment:
all this logic with random selection should be in the Test, not in the Page. it should be like
get all products
select random product
add it to cart
_cartPage.PlaceOrder()
Assert that correct product is on the order page